x-pack/plugins/observability_solution/apm/public/components/alerting/ui_components/alert_details_app_section/failed_transaction_chart.tsx

```diff
@@ -153,7 +154,7 @@ function FailedTransactionChart({
         <AlertActiveTimeRangeAnnotation
           alertStart={alertStart}
           alertEnd={alertEnd}
-          color={euiTheme.colors.backgroundBaseAccent}
+          color={transparentize(euiTheme.colors.danger, 0.2)}
```

Comment:
❓ Could we use `backgroundBaseDanger` (`danger10`) or `backgroundLightDanger` (`danger20`) here to prevent using `transparentize`?

Reply:
The issue here is the there are overlapping colors, where the original visual is not maintained if we don't use transparentize. Here's what it looks like before:
<img width="298" alt="Alert Threshold before" src="https://github.com/user-attachments/assets/455bbcee-cb98-44b8-997e-7dcb53092dbb" />

And here is what it looks like if transparentize isn't used: 
<img width="914" alt="Alert Threshold after" src="https://github.com/user-attachments/assets/f943b9e1-43e6-4fcd-a68f-7a30555e003a" />

Reply:
Ah gotcha, thanks for clarifying! 🙏